agent/rpm/pbench-agent.spec.j2

```diff
@@ -95,26 +95,35 @@ fi
 %posttrans
 
 %files
-%defattr(775,pbench,pbench,775)
+%defattr(664,root,root,775)
+/%{installdir}/ansible
 /%{installdir}/config
-
 /%{installdir}/lib
-/%{installdir}/ansible
-/%{installdir}/base
 
+/%{installdir}/base
 /%{installdir}/VERSION
 /%{installdir}/SEQNO
 /%{installdir}/SHA1
 /%{installdir}/profile
 
-%ghost %attr(0400,pbench,pbench) /%{installdir}/id_rsa
+%config(noreplace) %attr(0664,root,root) /%{installdir}/config/pbench-agent.cfg
 
-%defattr(775,pbench,pbench,775)
+%defattr(775,root,root,775)
 /%{installdir}/util-scripts
+%attr(664,root,root) /%{installdir}/util-scripts/README
 /%{installdir}/tool-scripts
+%attr(664,root,root) /%{installdir}/tool-scripts/perf.README
+%attr(664,root,root) /%{installdir}/tool-scripts/README
 /%{installdir}/bench-scripts
+%attr(664,root,root) /%{installdir}/bench-scripts/pbench-fio.md
+%attr(664,root,root) /%{installdir}/bench-scripts/pbench-uperf.md
+%attr(664,root,root) /%{installdir}/bench-scripts/postprocess/BenchPostprocess.pm
+%attr(664,root,root) /%{installdir}/bench-scripts/templates/fio.job
+%attr(664,root,root) /%{installdir}/bench-scripts/templates/fio-shared-fs.job
```

Comment:
Why set the default permissions for files to `775` and then override it to `664` on _every_ (file) entry?  (Why not just set the default to what you want and then omit the overrides?)

Also, it might be nice to segregate the files and directories, like you did at lines 98-107.

Reply:
There are more files in `/bench-scripts`, which will be subject to the default; only the non-executable files in the directory are overridden to exclude `xxx` permissions.

Although it's kinda interesting that the Perl module doesn't need execute? I suppose it's never run directly, only read by Perl; somehow it still feels odd. 🤷🏻‍♂️

Reply:
> There are more files in /bench-scripts, which will be subject to the default; only the non-executable files in the directory are overridden to exclude xxx permissions.

Ah.  Excellent.  Thank you.

> Although it's kinda interesting that the Perl module doesn't need execute? I suppose it's never run directly, only read by Perl

Yeah, it's not a "script" its an "include file".  🙂

Reply:
What is the concern here?

Reply:
I guess there is none.  🙂 
(I should have marked this as resolved after Dave's excellent answer.)